uaa/src/test/java/org/cloudfoundry/identity/uaa/scim/endpoints/ScimUserEndpointsAliasMockMvcTests.java

```diff
@@ -1600,7 +1615,7 @@ private static ScimUser buildScimUser(
         scimUser.setAliasZid(aliasZid);
         scimUser.setZoneId(zoneId);
 
-        scimUser.setUserName("john.doe");
+        scimUser.setUserName(TEST_USERNAME);
         scimUser.setName(new ScimUser.Name("John", "Doe"));
         scimUser.setPrimaryEmail("john.doe@example.com");
```

Comment:
not 100% sure if email should be TEST_USERNAME + "@example.com" to stay on safe side

Reply:
Good point, changing.